portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,17 +16,36 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import java.util.ArrayList;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    private String name;
+
+    private ArrayList<String> info = new ArrayList<String>() {{
```

Comment:
Prefer to use the least specific type as possible for the field type, so List instead of ArrayList. This allows you to easily change the implementation by only changing how the field is intiialized.

Reply:
Is it okay since the instructions says it has to be an array list.

Reply:
The concrete implementation used will still be an ArrayList but we can make the field's type less specific. Unless the method parameter requires an ArrayList I'd stick with List.

Reply:
Acknowledged

Reply:
Is it okay if I resolve these chnages in another branch ?